packages/expo-auth-session/src/AuthRequestHooks.ts

```diff
@@ -44,6 +44,12 @@ export function useLoadedAuthRequest(
 ): AuthRequest | null {
   const [request, setRequest] = useState<AuthRequest | null>(null);
   const scopeString = useMemo(() => config.scopes?.join(','), [config.scopes]);
+  const promptString = useMemo(() => {
+    if (config.prompt) {
+      return typeof config.prompt === 'string' ? config.prompt : config.prompt.join(',');
+    }
+    return undefined;
+  }, [config.prompt]);
```

Comment:
```suggestion
  const scopeString = config.scopes?.join(' ');
  const promptArray = config.prompt ? (Array.isArray(config.prompt) ? config.prompt : [config.prompt]) : undefined;
  const promptString = promptArray?.join(' ');
```

this

(1) fixes a potential bug where scopes and prompt are comma-delimited instead of space-delimited
(2) removes `useMemo`s which clutter the code for no benefit

You can also extract the logic of `promptString` into a separate function if you like (I'm not a huge fan of double-ternaries like this but for a inline suggestion this was the best).

Reply:
oh... (1) is my mistake... and I agree with (2). thanks for your suggestion! 

I will try to simplify that ternaries.

Reply:
extract with this commit 334d72a

Reply:
there was a misjudgment on my part - (1) isn't an issue because the value of `scopeString` and `promptString` is only used as a hook dependency, not in the actual request.